R/lat_bins_equal.R

```diff
@@ -0,0 +1,114 @@
+#' Generate equal-area latitudinal bins
+#'
+#' A function to generate approximately equal-area latitudinal bins for a
+#' user-specified given number of bins and latitudinal range. This approach is
+#' based on calculating the curved surface area of spherical segments bounded
+#' by two parallel disks.
+#'
+#' @param n \code{numeric}. A single numeric value defining the number of
```

Comment:
`n` should be an integer rather than just numeric, right?

Reply:
This is something I've thought about on a few occasions too (e.g. for `reps` in `bin_time()`). It is true that this number needs to be an integer (a whole number), but in terms of classes in R, this means the expected input would need to be the value followed by a L, e.g.
```
> is.integer(3)
[1] FALSE
> is.integer(3L)
[1] TRUE
```
Do we want to enforce this throughout the package or some class handling?

Reply:
Hmm. For me really this falls down to what the resultant behaviour is if someone tries to enter a non-integer number, so as long as the function handles this in a sensible way (that is clearly conveyed to the user), or throws an error saying the number should be an integer, it doesn't need to be formally handled as an integer by R.

Reply:
Agreed – I'll add error handling for this.